build.gradle

```diff
@@ -35,6 +35,11 @@ apply plugin: 'com.palantir.git-version'
 apply plugin: 'com.palantir.external-publish'
 apply plugin: 'com.palantir.baseline-java-versions'
 
+javaVersions {
+    libraryTarget = 11
+    runtime = 17
+}
```

Comment:
I would expect this to fail if atlasdb still sets `javaVersions {...` in `gradle/versions.gradle`.